src/Compilers/CSharp/Portable/Compilation/CSharpCompilation.cs

```diff
@@ -183,6 +192,34 @@ internal override CommonAnonymousTypeManager CommonAnonymousTypeManager
         /// </summary>
         internal bool IsPeVerifyCompatEnabled => LanguageVersion < LanguageVersion.CSharp7_2 || Feature("peverify-compat") != null;
 
+        internal bool RunNullableWalker
```

Comment:
>RunNullableWalker [](start = 22, length = 17)

Should this flag factor in whether the feature flag is set? In other words, if `NullableSemanticAnalysisEnabled` is true, then `RunNullableWalker` would return true too (regardless of compilation or syntax contexts).

Reply:
No, I don't think so. The intent is that when we want to turn the nullable semantic analysis on by default, this flag will influence `NullableSemanticAnalysisEnabled`, not the other way around.

---
In reply to: [296935067](https://github.com/dotnet/roslyn/pull/36714#discussion_r296935067) [](ancestors = 296935067)